e2e/utils/card-actions/remove.js

```diff
@@ -0,0 +1,15 @@
+const { dragCardTo } = require('../touch');
+
+const removeCard = async(browser, cardSelector) => {
+	const height = await browser.executeAsync(async(args, done) => {
+		const windowHeight = window.screen.height;
+
+		done(windowHeight);
+	}, '');
```

Comment:
**[💅🏽  nit-pick]** Maybe this utility is useful for this use case? https://webdriver.io/docs/api/browser/getWindowSize/

Reply:
I tried, it doesn't work for android :(